database/raft/Makefile

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
This file looks like it's for testing and debugging (and isn't required for normal building)? Please add a comment at the top that makes that clear.